README.md

```diff
@@ -19,6 +19,7 @@ Below is a listing of plugin versions and respective Velero versions that are co
 
 | Plugin Version | Velero Version |
 |----------------|----------------|
+| v1.7.x         | v1.11.x        |
```

Comment:
Please only keep the matrix no older than the Velero v1.7.x.

| Plugin Version | Velero Version |
|----------------|----------------|
| v1.7.x         | v1.11.x        |
| v1.6.x         | v1.10.x        |
| v1.5.x         | v1.9.x         |
| v1.4.x         | v1.8.x         |
| v1.3.x         | v1.7.x         |